### DISCUSSION TITLE
Github Pages CSS, JS, etc. generated by next.js is 404ing

### DISCUSSION THREAD

**Discussion by ifd3f** [community/community#23211] [Other Feature Feedback, Questions, & Ideas]
<p>My website (<a href="http://plenglin.github.io" rel="noopener nofollow ugc">plenglin.github.io</a>, cname’d to astrid.tech, <a href="https://github.com/Plenglin/plenglin.github.io" rel="noopener nofollow ugc">source code</a>) is giving a 404 when trying to fetch CSS. The site was statically generated using next.js.</p>
<p>Things I have tried:</p>
<ol>
<li>I have tried a hard refresh.</li>
<li>I have cleared my cookies.</li>
<li>I have looked at the Network tab in Firefox and compared it to my repo to ensure the CSS exists in the correct place. It does not.</li>
<li>I have served it locally using Python’s SimpleHTTPServer. The CSS works there.</li>
</ol>

**Comment by ifd3f** [comment]
<p>Because I’m not allowed to put more than 2 links in a post, here’s an example of something that’s not working:<br>
<a href="https://astrid.tech/_next/static/css/9712a16231e3eabbe829.css" rel="noopener nofollow ugc">https://astrid.tech/_next/static/css/9712a16231e3eabbe829.css</a> should correspond to <a href="https://github.com/Plenglin/plenglin.github.io/blob/c731cf80ca8746a27a03653b4091bdd1b97d61d8/_next/static/css/9712a16231e3eabbe829.css" rel="noopener nofollow ugc">this file</a> but the first link 404s.</p>

**Comment by ifd3f** [comment, marked as answer]
<p>I think I found the solution in <a href="https://github.community/t/css-not-being-applied-to-pabloocal-github-io/129597">this post</a>: adding a <code>.nojekyll</code> file to the root of my directory.</p>
<p><strong>EDIT:</strong> It did not work.<br>
<strong>EDIT 2:</strong> It did work, I just had to push a random change to force a rebuild of the site.</p>

**Reply by lukehatcher** [reply]
I was trying to get my astro site to work on GH pages and was also getting 404s for js/css.

Adding the `.nojekyll` to the root of my `gh-pages` branch worked
